src/loggers.jl

```diff
@@ -203,7 +211,7 @@ function get_logger(name="root")
 
     if !(haskey(_loggers, logger_name))
         parent = get_parent(logger_name)
-        _loggers[logger_name] = Logger(logger_name)
+        register(Logger(logger_name))
     end
 
     return _loggers[logger_name]
```

Comment:
Seems out of place now. Maybe `get_logger` should be used?

Reply:
This inside the `get_logger` function so that would result in infinite recursion.

Reply:
Carry on then :)